readme.md

```diff
@@ -771,6 +772,14 @@ Just type [`node.cool`](https://node.cool) to go here ✨
 - [nerds](https://github.com/SkyHacks/nerds) - Get data from nerdy topics like Harry Potter, Star Wars, and Pokémon.
 
 
+### Serialization
+
+- [msgpack-lite](https://github.com/kawanet/msgpack-lite) - Pure js implementation of the msgpack5 protocol.
+- [snappy](https://github.com/kesla/node-snappy) - Bindings for Google's serializer/ compressor.
+- [protobuf](https://github.com/dcodeIO/protobuf.js) - Implementation of Google's Protocol Buffers.
+- [compactr](https://github.com/compactr/compactr.js) - The fastest schema-based pure-js serializer.
```

Comment:
> The fastest

I couldn't find any benchmarks against Protocol Buffers and others.

Reply:
@fed135

Reply:
@sindresorhus Sorry, somehow I didn't get the notification. I since then updated the repo with some numbers. If the language is the issue, I can modify it in the PR.